kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -304,22 +304,19 @@ define([
                         case JOB_REQUESTS.LOGS:
                             this.sendBusMessage(JOB, jobId, RESPONSES.LOGS, {
                                 jobId,
-                                error: msgData.message,
-                            });
-                            this.sendBusMessage(JOB, jobId, 'job-log-deleted', {
-                                jobId,
-                                message: msgData.message,
+                                error: msgData,
                             });
                             break;
                         case JOB_REQUESTS.RETRY:
                             this.sendBusMessage(JOB, jobId, RESPONSES.RETRY, {
-                                jobId: jobId,
-                                error: msgData.message,
+                                jobId,
+                                error: msgData,
```

Comment:
send back the whole error object, just in case some part is of use (this error is not yet handled by the frontend)